### PR TITLE
Bug 1759969 Added variable for Importing insecure registries and blocking registries

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1759969

### CODE REVIEW

File: modules/images-configuration-insecure.adoc [hunk 1]
```diff
@@ -35,6 +35,8 @@ metadata:
   selfLink: /apis/config.openshift.io/v1/images/cluster
   uid: e34555da-78a9-11e9-b92b-06d6c7da38dc
 spec:
+  allowedRegistries:
```
Comment (xiuwang) [inline modules/images-configuration-insecure.adoc]: **allowedRegistries** should under spec.registrySources, and the vlaues should be in a list without domainName keyword.
e.g. 
spec:
  registrySources:
    allowedRegistries:
    - quay.io
See the definition https://github.com/openshift/api/blob/master/config/v1/types_image.go#L101-L114

Comment (xiuwang) [reply modules/images-configuration-insecure.adoc]: Should add explain for allowedRegistries too.

Comment (bmcelvee) [reply modules/images-configuration-insecure.adoc]: Thank you! Updated.

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review. Thanks!

File: modules/images-configuration-insecure.adoc [hunk 2]
```diff
@@ -45,15 +45,18 @@ spec:
...
+    allowedRegistries:
+      - domainName: quay.io <3>
 status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ----
 <1> Specify an insecure registry.
 <2> Specify any registries that should be blacklisted for image pull and push actions. All other
-registries are allowed.
+registries are allowed. Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
+<3> Specify registries permitted for image pull and push actions. All other registries are denied. Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
```
Comment (bobfuru) [inline modules/images-configuration-insecure.adoc]: s/registries permitted/registries that should be permitted/
Using the same wording construct for `<2>` and `<3>` makes it easier to understand what each is doing.

**Review by bobfuru — COMMENTED** [review]
One suggestion and otherwise LGTM!

File: modules/images-configuration-insecure.adoc [hunk 3]
```diff
@@ -45,15 +45,18 @@ spec:
     - insecure.com
     blockedRegistries:<2>
     - untrusted.com
+    allowedRegistries:
+      - domainName: quay.io <3>
```
Comment (xiuwang) [inline modules/images-configuration-insecure.adoc]: No domainName parameter for allowedRegistries
should be
```allowedRegistries:
    - quay.io
```

File: modules/images-configuration-insecure.adoc [hunk 4]
```diff
@@ -45,15 +45,18 @@ spec:
     - insecure.com
     blockedRegistries:<2>
     - untrusted.com
+    allowedRegistries:
+      - quay.io <3>
```
Comment (xiuwang) [inline modules/images-configuration-insecure.adoc]: `- quay.io` should align with allowed, like `blockedRegistries` does

**Comment by xiuwang** [comment]
LGTM

**Pull request merged**